packages/explorer/src/components/ui/input.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { cn } from "@/lib/utils";
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
```

Comment:
curious about this pattern vs
```
export type Props =  React.InputHTMLAttributes<HTMLInputElement>
```

Reply:
Good point, I'm not aware of why it's done this way (default shadcn/ui exported component), I'll simplify it with your suggestion.

Reply:
so looking at other shadcn components it seems like it's just a pattern that makes it easy to extend the type, such as in `Button` component:

```typescript
export interface ButtonProps
  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
    VariantProps<typeof buttonVariants> {
  asChild?: boolean;
}
```

However, I don't see why it's needed given there are no additional props we need to add so changed to which is consistent to how types are declared in other components:

```typescript
const Input = React.forwardRef<
  HTMLInputElement,
  React.InputHTMLAttributes<HTMLInputElement>
>(({ className, type, ...props }, ref) => {
```